src/dreammaker/builtins.rs

```diff
@@ -1060,6 +1088,35 @@ pub fn register_builtins(tree: &mut ObjectTree) {
 
         regex/proc/Find_char(text, start, end);
         regex/proc/Replace_char(text, rep, start, end);
+
+        // 514 stuff
+
+        generator;
+        generator/proc/Rand();
+        generator/proc/Turn(a);
+
+        particles;
```

Comment:
these are annotated as `A datum (not derived from /atom)` which causes the ref lookup for this to be weird - something to investigate

...so it looks like there's also a var on movable atoms named the same thing 🤦‍♂️ 

also there's a reference error related: http://www.byond.com/forum/post/2652272